privacyidea/lib/tokens/hotptoken.py

```diff
@@ -493,7 +493,7 @@ def resync(self, otp1, otp2, options=None):
             return ret
 
         ret = True
-        self.inc_otp_counter(counter + 1, True)
+        self.inc_otp_counter(counter + 1, reset=True)
```

Comment:
self.inc_otp_counter(reset=True)

should also work.

Reply:
Hmm that's right. Actually, even `self.inc_otp_counter()` would work, I just didn't want to touch more code than necessary :)

Reply:
No I am. Wrong. I could be the not latest OTP value. Thanks!